code/modules/reagents/dispenser/cartridge_presets.dm

```diff
@@ -94,7 +94,10 @@
 	bicaridine	spawn_reagent = "bicaridine"
 	hyperzine	spawn_reagent = "hyperzine"
 	rezadone	spawn_reagent = "rezadone"
-	spaceacillin	spawn_reagent = "spaceacillin"
+	thetamivir	spawn_reagent = "thetamivir"
+	methicillin spawn_reagent = "methicillin"

```

Comment:
indentation AAAAAAAAAAA

Reply:
works for me man https://i.imgur.com/PMMrbNI.png

Reply:
okay wait i used a space and not a tab